client/src/totalRatings.jsx

```diff
@@ -6,14 +6,16 @@ import { faStar } from '@fortawesome/free-solid-svg-icons';
 const TotalRatings = ({ total, average }) => {
   const stars = [];
   for (let index = 1; index <= 5; index += 1) {
-    const color = index <= average ? 'gold' : 'grey';
-    stars[index] = <FontAwesomeIcon key={`star-${index}`} icon={faStar} color={color} />;
+    const color = index <= average ? '#FDC038' : '#A5A8AB';
+    stars[index] = (
+      <FontAwesomeIcon className="star" key={`star-${index}`} icon={faStar} color={color} />
+    );
   }
   return (
-    <div color="blue">
+    <div id="total-ratings-container">
       {stars}
-      &emsp;(
```

Comment:
What does 'emsp' mean?

Reply:
It's a way to add large spaces / tabs to text in HTML divs (https://stackoverflow.com/questions/28058735/thinsp-ensp-emsp-alternatives-in-safari-browser)